docker-compose.yaml

```
@@ -1,15 +1,15 @@
 version: "3"
 services:
   db:
-    image: postgres:latest
+    image: postgres:alpine
```

```suggestion
    image: postgres:14-alpine
```

See https://github.com/projectcaluma/caluma/pull/1704/commits/29614e901c51c9a73d417a3174507b2cec62049d